config/jobs/kubernetes/sig-scalability/sig-scalability-presubmit-jobs.yaml

```diff
@@ -48,7 +48,9 @@ presubmits:
         - --test-cmd-args=--report-dir=/workspace/_artifacts
         - --test-cmd-args=--testconfig=testing/density/config.yaml
         - --test-cmd-args=--testconfig=testing/load/config.yaml
-        - --test-cmd-args=--testconfig=testing/access-tokens/config.yaml
+        # 'access-tokens' test is explicitly missing from this presubmit, as it is a very targeted test.
```

Comment:
I'm not sure this comment really gives us much. I would remove it.

Reply:
We should mark here with this test is different than CI test - otherwise we would need to transfer this knowledge in person in case a new engineer joins oncall. Could you suggest a rephrase of this comment?

Reply:
I actually claim we will be diverging more and more in the future.
I would like to run just load as part of presubmit and more tests in postsubmit (there will be more and more tests coming).
Mentioning all of them everywhere doesn't make much sense.

Reply:
I do not want to lose a relationship between tests and explanation why there are differences. Some comment here to document can really help.

I'm okay with changing this comment to a more verbose version of 'presubmit test is a lighweight version of periodic test to save time'.

Reply:
Let's comment at the job level, not here then.

Reply:
Done.